src/group.h

```diff
@@ -202,6 +202,10 @@ static void secp256k1_ge_from_bytes(secp256k1_ge *r, const unsigned char *buf);
  */
 static int secp256k1_ge_is_in_correct_subgroup(const secp256k1_ge* ge);
 
+static void secp256k1_point_save_ext(unsigned char *data, secp256k1_ge *ge);
+
+static void secp256k1_point_load_ext(secp256k1_ge *ge, const unsigned char *data);
```

Comment:
The libsecp repo defines these functions [using different names](https://github.com/bitcoin-core/secp256k1/pull/1479/files#diff-28d9b92418cbddbe487fe84847993cc5da0b096363c83631a525cff25189752cR179-R191):
- `point_save_ext` -> `ge_to_bytes_ext`
- `point_load_ext` -> `ge_from_bytes_ext`

Reply:
https://github.com/BlockstreamResearch/secp256k1-zkp/pull/278/commits/9fcaf25be7ccd96f6c4da9b292daf6ff4cf10e3b